src/models/event.js

```diff
@@ -689,6 +694,10 @@ utils.extend(MatrixEvent.prototype, {
         return this._forwardingCurve25519KeyChain;
     },
 
+    isUntrusted: function() {
```

Comment:
At the backup session level, the general word "untrusted" is perhaps fairly clear, but at the event level, I am worried that are too many different kinds of trust that we should be a bit more specific... Also, is it possible to write this in terms of a positive word (to avoid double negatives)? Perhaps `isKeySourceTrusted`?